near-sdk/src/collections/tree_map.rs

```diff
@@ -173,6 +258,25 @@ where
     ///
     /// Panics if range start > end.
     /// Panics if range start == end and both bounds are Excluded.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// let mut map: TreeMap<u32, u32> = TreeMap::new(b"m".to_vec());
+    /// let one: Vec<u32> = vec![10, 20, 30, 40, 50];
+    /// let two: Vec<u32> = vec![45, 35, 25, 15, 
```

Comment:
You're missing a closing brace here. Wondering why the need for two vectors if just indicating which keys to insert